README.md

```diff
@@ -8,6 +8,10 @@ An add-on to let you snooze your tabs for a while.
   * Starts a file watcher to rebuild on changes
   * Launches Firefox Dev Edition with the extension, reloaded on changes
 
+* If youʼre on Windows, youʼll need to use `npm start-win`
```

Comment:
This should be `npm run start-win`, because it's not one of the "built-in" npm script short cuts.